README.md

```diff
@@ -1,90 +1,68 @@
 Zydis Pascal Bindings
 =====================
 
-[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT) [![Gitter](https://badges.gitter.im/zyantific/zyan-disassembler-engine.svg)](https://gitter.im/zyantific/zydis?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=body_badge) [![Discord](https://img.shields.io/discord/390136917779415060.svg)](https://discordapp.com/channels/390136917779415060/390138781313007626) 
+[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
 
 Pascal language bindings for the [Zydis library](https://github.com/zyantific/zydis), a fast and lightweight x86/x86-64 disassembler.
 
-## Readme
-The Zydis pascal bindings support both dynamic and static linkage. The required binaries for Windows can be acquired in the [Releases](https://github.com/zyantific/zydis-pascal/releases) section.
+## TODO
+This repo still in development & Supports only FPC for now
 
-#### Dynamic
-- Extract the `Zydis32.dll` or `Zydis64.dll` from the `Bin32`/`Bin64` folder to the directory that contains your application or install the library to your `C:\Windows\System32\` (64-bit) / `C:\Windows\SysWOW64\` (32-bit) directory
-- Enable the `{$DEFINE ZYDIS_DYNAMIC_LINK}` compiler-directive (enabled by default) in `Zydis.pas`
-
-#### Static
-- Extract the complete `Bin32`/`Bin64` folder to the root of the pascal bindings directory
-- Disable the `{$DEFINE ZYDIS_DYNAMIC_LINK}` compiler-directive in `Zydis.pas`
```

Comment:
I'm not a super huge fan of including binary files in a Git repository, but we can consider going that route, if we feel like this would lower the burden of using the bindings.

Anyways, I think that part of the readme should stay. At least the part about dynamic linking might be important to some users - regardless of whether we bundle the static binaries in the repo or not.

Reply:
I think Delphi is still mainly used on Windows, but I'm not sure how the situation is for FPC. It easily gets compilcated when we have to bundle a set of X86(_64) and ARM(64) binaries for different OSs like Windows, Linux, FreeBSD, etc. This was the original reason for not including the binaries in the repo.

Reply:
I think we could create a GitHub Actions release workflow that uses GCC or Clang to cross-compile Zydis for different target architectures and operating systems in different formats (`.a`, `.obj`, `.dll`, ...).

Reply:
I agree that `dynamic linking` is essential for many users.
For me, I only use Delphi for User Interface projects, but Pascal, I do use it for all platforms.
I do have Delphi, but I'll do most of the work on FPC first and then test it with Delphi.

GitHub Actions release workflow would be great for auto-bundling Zydis binaries.